datafusion/datasource-parquet/src/reader.rs

```diff
@@ -212,7 +212,7 @@ impl ParquetFileReaderFactory for CachedParquetFileReaderFactory {
 /// Implements [`AsyncFileReader`] for a Parquet file in object storage. Reads the file metadata
 /// from the [`FileMetadataCache`], if available, otherwise reads it directly from the file and then
 /// updates the cache.
-pub(crate) struct CachedParquetFileReader {
+pub struct CachedParquetFileReader {
```

Comment:
Likewise, I locally reverted these changes to visibility and everything seems to have compiled just fine